podcasts/Whats New/WhatsNewView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct WhatsNewView: View {
+    @EnvironmentObject var theme: Theme
+
+    let announcement: WhatsNewAnnouncement
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ZStack(alignment: .top) {
+                Rectangle()
+                    .frame(height: 195)
+
+                HStack {
+                    Spacer()
+                    Button {
+                        NavigationManager.sharedManager.dismissPresentedViewController()
+                    } label: {
+                        ZStack {
+                            Image("close")
+                                .foregroundColor(.white)
+                        }
+                        .frame(width: 44, height: 44)
+                    }
+                }
+            }
+            Text(announcement.title)
+                .font(style: .title3, weight: .bold)
+                .padding(.horizontal)
+                .padding(.top)
+                .foregroundColor(theme.primaryText01)
+            Text(announcement.message)
+                .font(style: .subheadline)
+                .foregroundColor(theme.secondaryText02)
+                .padding(.horizontal)
+                .padding(.bottom)
+            Button("Enable it") {}
+                .buttonStyle(RoundedDarkButton(theme: theme))
+                .padding(.horizontal)
+                .padding(.bottom)
+        }
+        .frame(width: 340)
+        .background(.white)
+        .cornerRadius(5)
```

Comment:
[nit] On the smallest device we support (iPod Touch) the dialog is clipped on edges

<img width="120" src="https://github.com/Automattic/pocket-casts-ios/assets/793774/e5ceb206-d60c-4453-ae39-f8dd7b3171e1">

We can make this more flexible by using a min/maxWidth and adding padding at the end. 

```suggestion
        .frame(minWidth: 300, maxWidth: 340)
        .background(.white)
        .cornerRadius(5)
        .padding()
```